plugin/src/test/java/org/opensearch/ml/task/TaskRunnerTests.java

```diff
@@ -139,8 +138,8 @@ public void testRun_CircuitBreakerOpen() {
         TransportService transportService = mock(TransportService.class);
         ActionListener listener = mock(ActionListener.class);
         MLTaskRequest request = new MLTaskRequest(false);
-        expectThrows(MLLimitExceededException.class, () -> mlTaskRunner.run(FunctionName.REMOTE, request, transportService, listener));
+        mlTaskRunner.run(FunctionName.REMOTE, request, transportService, listener);
         Long value = (Long) mlStats.getStat(MLNodeLevelStat.ML_CIRCUIT_BREAKER_TRIGGER_COUNT).getValue();
-        assertEquals(1L, value.longValue());
+        assertEquals(0L, value.longValue());
```

Comment:
Name of the test is `testRun_CircuitBreakerOpen()` and now we are asserting there's no circuit breaker open. May be we can refactor the name at least?